Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -305,3 +310,10 @@ extension PartialIso where A == (String?, Int?), B == Pricing {
     })
   }
 }
+
+private let isTest: Router<Bool?> =
+  formField("live", .string).map(isPresent >>> negate >>> Optional.iso.some)
+    <|> formField("test", .string).map(isPresent >>> Optional.iso.some)
+
+private let isPresent = PartialIso<String, Bool>(apply: const(true), unapply: { $0 ? "" : nil })
+private let negate = PartialIso<Bool, Bool>(apply: (!), unapply: (!))
```

Comment:
and this stuff down here ain't great either, but maybe this is just how it has to be. I wanna parse out the `live` and `test` form fields, and map their presence to either `true` or `false`.

Reply:
Do we wanna use a `enum: String` type for this at the least?

Reply:
intersting, but not sure it would work. it's not that a single value is coming back as either `live` or `test`, it's that either a `live` value comes back _or_ a `test` value. it's just cause of how html forms work. I have two submit buttons each with different values. open to other ways of doing this too, it's just want I came up with so that I could have a single form for both the test email and live emails.